src/compiler/scala/reflect/macros/runtime/JavaReflectionRuntimes.scala

```diff
@@ -39,7 +39,7 @@ trait JavaReflectionRuntimes {
           case _ => false
         }
 
-        val Array(bundleCtor) = implClass.getConstructors.filter(isBundleCtor)
+        val Array(bundleCtor) = (implClass.getConstructors.filter(isBundleCtor): @unchecked)
```

Comment:
unnecessary parens

Reply:
Yeah, I ended up understanding that after a while... 😄   Will fix.